assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserDimensionsPieChart.js

```diff
@@ -99,10 +100,43 @@ export default function UserDimensionsPieChart( {
 			}
 		};
 
+		const closeToolTip = () =>
+			setValues( {
+				[ UI_DIMENSION_VALUE ]: '',
+				[ UI_DIMENSION_COLOR ]: '',
+				[ UI_ACTIVE_ROW_INDEX ]: null,
+			} );
+
+		const tooltipIsOpen = !! currentContainerRef.querySelector(
+			'.google-visualization-tooltip'
+		);
```

Comment:
@johnPhillips we need to check it in `onEscape` and `onExitClick` callbacks:

```javascript
const onEscape = ( event = {} ) => {
    const tooltipIsOpen = !! currentContainerRef.querySelector( '.google-visualization-tooltip' );
    if ( tooltipIsOpen && event?.keyCode === ESCAPE ) {
        ...
    }
};

const onExitClick = ( event = {} ) => {
    const tooltipIsOpen = !! currentContainerRef.querySelector( '.google-visualization-tooltip' );
    if ( tooltipIsOpen && ... ) {
        ...
    }
};
```

Reply:
Of course, sorry.